src/main/java/TicTacToe/WinLogic.java

```diff
@@ -0,0 +1,40 @@
+package TicTacToe;
+
+public class WinLogic {
+
+    public boolean checkWin() {
+        char symbol;
+
+        for (int i = 0; i < Board.board.length; i++) {
```

Comment:
1. Wyciągnij do funkcji np. checkSameSymbolInColumns(rowNumber, Symbol symbol)
2. W jednej pętli sprawdzasz i czy są 3 te same sybole w rzędach i kolumnach. Zrób to w dwóch pętlach